assets/src/scripts/components/dailyValueHydrograph/selectors/legend-data.spec.js

```diff
@@ -0,0 +1,98 @@
+import {select, selectAll} from 'd3-selection';
+import {drawTimeSeriesLegend} from '../legend';
```

Comment:
You should not have a test for drawTimeSeriesLegend in here. This should be in a test spec for ../legend.spec.js.

Reply:
I would like the test for drawTimeSeriesLegend to be moved from this file to ../legend.spec.js.